src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -26,6 +31,7 @@ export const Tasks: React.FC = () => {
   }
   return (
     <>
+      { taskCount !== "" && (<P2>{data.patchTasks.length}/{taskCount} tasks</P2>)}
```

Comment:
doing just `taskCount && (render code)` should be ok here and will also check `taskCount` for truthiness in case it's ever undefined or null for some reason